CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Rename `Clinical significanc` to `Germline classification` in ClinVar submissions exported files
 - Rename `Comment on clinical significance` to `Comment on classification` in ClinVar submissions exported files
 - Show matching partial causatives on variant page
+- Matching causatives query
```

Comment:
Probably add a slightly meatier explanation, including more exactly what was wrong and perhaps a date-range as well. The functionality is mostly for extra safety and speed, there is a world where we kind of rely on it to pick up less obvious phenotypes.